tic_tac_toe/game.py

```diff
@@ -24,7 +30,10 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
+    if isinstance(position, tuple) and len(position) <= 2 and len(position) >= 0 and position[0] >= 0 and position[0] <= 2 and position[1] >= 0 and position[1] <= 2:
```

Comment:
This is a really long line. You could break it up into multiple lines by wrapping everything after the if in parenthesis or by using `\`
Examples:
```
if True and True:
    pass
if (True and
    True):
    pass
if True and\
   True:
    pass
```
All do the same thing. According to Python [PEP8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) (style guidelines) line width should be limited to 79 characters for readability.